src/scale.js

```diff
@@ -0,0 +1,81 @@
+/* global WebAnimationsPolyfillExtension internalScope */
+
+(function () {
+  function isNumeric (number) {
+    return !isNaN(number);
+  }
+
+  function addDefaultAxis (values) {
+    if (values.length === 1) { // x-axis was specified
+      values.push(1, 1);
+    } else if (values.length === 2) { // x and y-axis were specified
+      values.push(1);
+    }
+    return values;
+  }
+
+  function scaleParse (input) {
+    /* According to spec:
+      https://drafts.csswg.org/css-transforms-2/#propdef-scale
+      unspecified scales default to 1
+    */
+    if (input === undefined) {
+      return null;
+    } else if (input === 'none') {
+      return [1, 1, 1];
+    }
+
+    var values = input.split(/\s+/);
+    var numValues = values.length;
+    if (numValues < 1 || numValues > 3) {
+      return undefined;
```

Comment:
I probably wasn't clear when I said parse should return undefined. I didn't mean for the toTransform behaviour to change, just for the extension API parse hook to catch the InvalidArgument errors and return undefined instead.